open-company/open-company-api
# History count in board response

https://trello.com/c/knMG0hRu

This provides a `count` property on the res `collection` of each entry in the board response so the UI knows how many entries there are.

Uses this PR from oc.lib: https://github.com/open-company/open-company-lib/pull/4

To test:

* Import 18F to your team.
* Look at the `/boards/all-hands` response. Each entry should have 1 on its `count` property of the res `collection` link except for the `press` topic which has 2. 
* Now add an entry for 1 of the other topics, reload the board response and check that it has 2 as its `count` now.
* Add another entry to `press`, check that it now has 3. 
* Remove an entry from `press`, check that it now has 2.
* Review the code
* Merge
* Deploy
* Kiss your ferrets
